htdocs/install/mysql/tables/llx_element_type.sql

```diff
@@ -0,0 +1,33 @@
+-- ============================================================================
+-- Copyright (C) 2008-2011	Laurent Destailleur	<eldy@users.sourceforge.net>
+-- Copyright (C) 2011		Regis Houssin		<eldy@users.sourceforge.net>
+--
+-- This program is free software; you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation; either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with this program. If not, see <https://www.gnu.org/licenses/>.
+--
+-- ============================================================================
+-- Table used for relations between elements of different types:
+-- invoice-propal, propal-order, etc...
+-- ============================================================================
+
+create table llx_element_type
+(
+  rowid           	integer AUTO_INCREMENT PRIMARY KEY,  
+  fk_element		integer NOT NULL,
+  element_type		varchar(64) NOT NULL,
+  element		    varchar(32) NOT NULL,
+  module_name		varchar(31) NOT NULL,
+  class_path		text NOT NULL,
```

Comment:
D'un point de vue sémantique, je m'attends à ce que `path` contienne un chemin complet (y compris le nom de fichier)
```suggestion
  class_dir		text NOT NULL,
```